src/interfaces/books.ts

```diff
@@ -0,0 +1,14 @@
+import { Category, Tags } from '../types/books'
+
+export interface IBook {
+    id: number,
+    title: string,
+    author: string,
+    publisher?: string,
+    publication_date?: string,
+    isbn?: string,
+    description: string,
+    cover_image: string,
```

Comment:
Nit - should there be a fancier image data structure? One that has the `width` and `height`, maybe also an `alt`, for example?

Reply:
Sure, we can add this in the upcoming issues, while detailing the layout of books. I'll prefer to keep it as it is, for now.